<launch>

<!--  <node name = "gripper_env_node"-->
<!--        pkg = "gripper_minimal"-->
<!--        type = "gripper_env.py"-->
<!--        respawn = "false"-->
<!--        output = "screen"-->
<!--  />-->

	<node name="gripper_qlearn"
				pkg="gripper_minimal"
				type="start_qlearning.py"
				respawn="false"
				output="screen"
	/>

</launch>
